.gitignore

```diff
@@ -25,3 +25,9 @@ Thumbs.db
 # repos / submodules
 .git
 .svn
+
+public/bundle.js
+public/bundle.js.map
+secrets.js
+yarn.lock
```

Comment:
Don't ignore lockfiles. Otherwise you're making your deployments nondeterministic.